WordPress/src/main/res/layout/activity_log_type_filter_item.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Minor: This file can be reformatted.

Reply:
Fixed in cdd2759f98e31aefebe936d5b6e23a2039b1badc